src/sidebar/services/frame-sync.js

```diff
@@ -220,26 +220,4 @@ export default function FrameSync(annotationsService, bridge, store) {
     setupSyncToFrame();
     setupSyncFromFrame();
   };
-
-  /**
-   * Focus annotations with the given $tags.
-   *
-   * This is used to indicate the highlight in the document that corresponds to
-   * a given annotation in the sidebar.
-   *
-   * @param {string[]} tags - annotation $tags
-   */
-  this.focusAnnotations = function (tags) {
```

Comment:
Given that this method encapsulated two actions—a store action and a `bridge.call`, it would have been nice to extract it to somewhere else instead of re-implementing it in the two places it's used. But I didn't see a great place to stick it. I didn't want to create a whole new service for it. It doesn't really fit in the `annotationsService`, as that service pertains to CRUD-like actions on a single annotation. It doesn't fit in any other service obviously. Because of its reliance on the `store`, it can't be in a `util` module. So...any ideas welcome. Or I can leave it for now.